ui/docs/reference/core/worker-manager/launch-configurations.mdx

```diff
@@ -0,0 +1,72 @@
+---
+title: Worker-Manager - Launch Configurations
+order: 100
+---
+
+# Worker-Manager - Launch Configurations
+
+Launch configurations define how Worker-Manager should create and manage cloud instances for worker pools.
+This feature is supported by the following cloud providers:
+- [AWS Provider](/docs/reference/core/worker-manager/aws.mdx)
+- [Azure Provider](/docs/reference/core/worker-manager/azure.mdx)
+- [Google Provider](/docs/reference/core/worker-manager/google.mdx)
+
+## Configuration
+
+Each launch configuration _may_ include a `workerManager` object that controls how instances are provisioned:
+
+```yaml
+workerManager:
+  launchConfigId: 'unique-config-id'      # Optional: Unique identifier for this configuration
+  capacityPerInstance: 1                  # Number of workers per instance
+  initialWeight: 0.5                      # Float between 0 and 1 (default: 1)
```

Comment:
```suggestion
  initialWeight: 0.5                      # Float between 0 and 1 (default: 0.5)
```

Reply:
Apologies! I see default _should be_ 1, but in the schema, you mention how the default is `0.5`, knew I got that value from somewhere 😅

Reply:
brilliant, thanks for spotting this! indeed, constants.yml should be saying that `1` is the default, correcting it